test/test_qt_utils.py

```diff
@@ -26,7 +26,7 @@
 @pytest.mark.unit
 def test_pauli_basis() -> None:
     """Testing dimensions of Pauli basis"""
-    dims = np.random.randint(2, 5, np.random.randint(1, 5))
+    dims = np.random.randint(2, 5, np.random.randint(2, 4))
```

Comment:
We have these hardcoded constants in several places throughout this file and that is difficult to maintain (and update). Can we move all of these to global constants mentioned at the top of the file? This ensures that there is a single point of update when we decide to modify this next time. So something like `RANDOM_INT_DIM = [. . .]` for all the edits you made in this file now.

Reply:
I don't know if they should all be the same. Comments welcome.

Reply:
Yes they could be the same. There is not reason for having different values. I just used dimensions that made sense and were not too large for the tests to run forever.

Reply:
In that case, I recommend we use the same dimensions for all the tests, define it in a global constant at the top of the file and provide a note on the choice of the dimension in the module level docstring. Maybe like this (but not so much explanation or verbosity):
https://github.com/q-optimize/c3/blob/6ee8d75159e04b1ac8b4e97f78cf84755000ddd3/test/test_model_learning.py#L1-L22

Reply:
Additionally, let's keep the dimensions as low as possible, to the smallest possible configuration. This is causing far too many OOM fails in our CI.